rallyrolebot/cogs/channel_cog.py

```diff
@@ -109,3 +109,29 @@ async def get_channel_mappings(self, ctx):
                 ]
             )
         )
+
+    @commands.command(name="set_purchase_message", help="Change the $purchase message")
+    @validation.owner_or_permissions(administrator=True)
+    async def set_purchase_message(self, ctx, message):
+        data.set_purchase_message(ctx.guild.id, ctx.message.content.split(' ', 1)[1])
+
+    @commands.command(name="purchase", help="Learn how you can purchase")
+    async def purchase(self, ctx):
+        message = data.get_purchase_message(ctx.guild.id)
+        if message is not None:
+            await ctx.send(message)
+        else:
+            await ctx.send(DEFAULT_PURCHASE_MESSAGE)
+
+    @commands.command(name="set_donate_message", help="Change the $donate message")
+    @validation.owner_or_permissions(administrator=True)
+    async def set_donate_message(self, ctx, message):
+        data.set_donate_message(ctx.guild.id, ctx.message.content.split(' ', 1)[1])
```

Comment:
Use discord.py keyword arguments for a cleaner function
```suggestion
    async def set_donate_message(self, ctx, *, message):
        data.set_donate_message(ctx.guild.id, message)
```

Reply:
Done.